static/docs/user-guide/experiments/mixed.md

```diff
@@ -0,0 +1,26 @@
+# How to Manage Experiments by Several Methods
```

Comment:
Mix Methods to Manage Experiments?

Reply:
That's fine too.

Reply:
👍

Reply:
Maybe ** How to Manage Experiments with a Mix of Methods**
or use the word "Combination", as in the intro paragraph.